src/Microsoft.NetCore.Analyzers/UnitTests/Security/ApprovedCipherModeTests.cs

```diff
@@ -148,14 +154,14 @@ End Sub
             );
         }
 
-        protected override DiagnosticAnalyzer GetBasicDiagnosticAnalyzer()
-        {
-            return new ApprovedCipherModeAnalyzer();
-        }
+        private static DiagnosticResult GetCSharpResultAt(int line, int column, params string[] arguments)
```

Comment:
Yeah seems like you are defining this in almost all the test files. Might make sense to have these as helper methods, unless they are causing a lot of churn on your side - if so it is fine to leave it as a follow-up cleanup item.

Reply:
Probably good to have these as internal static helper methods on VerifyCS and VerifyVB?

Reply:
See #3088